engine/src/main/java/io/seldon/engine/api/rest/RestClientController.java

```diff
@@ -137,17 +140,17 @@ String unpause() {
     public ResponseEntity<String> predictions_json(RequestEntity<String> requestEntity)
 	{
 		logger.debug("Received predict request");
-		Scope tracingScope = null;
+		Span tracingSpan = null;
 		if (tracingProvider.isActive())
-			tracingScope = tracingProvider.getTracer().buildSpan("/api/v0.1/predictions").startActive(true);
+			tracingSpan = tracingProvider.getTracer().buildSpan("/api/v0.1/predictions").start();
```

Comment:
Has the API for tracing changed?

Reply:
Yes, it did. They've removed all methods which assumed an automatic start/finish of a `Span` linked to a `Scope`. For example, in this case, `startActive()` would return a `Scope` which already had a `Span` implicitly created. This `Span` would then get finished automatically when the `Scope` got destroyed.

Instead, you are now forced to handle manually the start and finish of each `Span`.